visualization/app/codeCharta/util/scenarioHelper.ts

```diff
@@ -138,8 +130,9 @@ export class ScenarioHelper {
 		if (ccLocalStorage) {
 			return new Map(ccLocalStorage.scenarios)
 		}
-		this.setScenariosToLocalStorage(this.getPreLoadScenarios())
-		return this.getPreLoadScenarios()
+		const scenarios = this.getPreLoadScenarios()
+		this.setScenariosToLocalStorage(scenarios)
+		return scenarios
```

Comment:
Just curious why this has been changed.
Is this just coding style or is there actually a valid reason to not call a method as a parameter of another method?

Reply:
It's only run once instead of twice.